docs/faq.md

```diff
@@ -91,7 +91,7 @@ browser.takeScreenshot().then(function(png) {
 ```
 
 The method to take a screenshot automatically on failure would depend on the type of failure.
-* For failures of entire specs (such as timeout or an expectation within the spec failed), you can add a reporter as such:
+* For failures of entire specs (such as timeout or an expectation within the spec failed), you can add a reporter as such (this is jasmine 1.3 syntax):
```

Comment:
I think this would be better as a comment inside the code block instead of as a parenthetical